<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderManage</class>
 <widget class="QWidget" name="ReaderManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leSearchReader">
       <property name="placeholderText">
        <string>请输入读者ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearchReader">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewReaders">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAddReader">
       <property name="text">
        <string>添加读者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteReader">
       <property name="text">
        <string>删除读者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefreshReader">
       <property name="text">
        <string>刷新列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
